### ISSUE TITLE
TypeError: Cannot read property 'slider' of undefined

### ISSUE THREAD

**Issue by agungsb** [brownieboy/react-bootstrap-slider#16]
Hi,

I need help here. I got the following error when using this plugin.

```
TypeError: Cannot read property 'slider' of undefined
   at /Applications/XAMPP/xamppfiles/htdocs/kumparan-cms-jared/node_modules/bootstrap-slider/dist/bootstrap-slider.js:1676:14
   at /Applications/XAMPP/xamppfiles/htdocs/kumparan-cms-jared/node_modules/bootstrap-slider/dist/bootstrap-slider.js:1691:6
   at /Applications/XAMPP/xamppfiles/htdocs/kumparan-cms-jared/node_modules/bootstrap-slider/dist/bootstrap-slider.js:1693:4
   at NAMESPACE_MAIN (/Applications/XAMPP/xamppfiles/htdocs/kumparan-cms-jared/node_modules/bootstrap-slider/dist/bootstrap-slider.js:51:20)
   at Object.<anonymous> (/Applications/XAMPP/xamppfiles/htdocs/kumparan-cms-jared/node_modules/bootstrap-slider/dist/bootstrap-slider.js:55:3)
   at Module._compile (module.js:541:32)
   at Module._extensions..js (module.js:550:10)
   at Object.require.extensions.(anonymous function) [as .js] (/Applications/XAMPP/xamppfiles/htdocs/kumparan-cms-jared/node_modules/babel-register/lib/node.js:154:7)
   at Module.load (module.js:458:32)
   at tryModuleLoad (module.js:417:12)
```

Can you help me to solve this? Thanks

**Comment by brownieboy** [comment]
Try posting some source code so we can see how you're using the plugin.

**Comment by brownieboy** [comment]
Unable to reproduce.  Further information that I requested was not supplied, so I'm closing this.  Post some code and I'll reopen.

**Comment by isaachinman** [comment]
For what it's worth, I also encountered this error when importing ReactBootstrapSlider in SSR. I think it has something to do with node vs client, but I dug around a bit and couldn't find anything obvious, except [this](https://github.com/brownieboy/react-bootstrap-slider/blob/master/dist/react-bootstrap-slider.js#L19). Anyways I have already abandoned the component, but just wanted to let you know that you have a critical error preventing universal/SSR use.
